draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -1045,23 +983,22 @@ A registration MUST include the following fields:
 * Pointer to specification text
 
 Values to be added to this name space require Expert Review (see
-{{!RFC5226}}, Section 4.1).
+{{!RFC5226}}, Section 4.1).  Apart from the initial contents, the name
+MUST NOT start with "key".
 
 ### Initial contents
 
 The "Service Binding (SVCB) Parameter Registry" shall initially
 be populated with the registrations below:
 
-| SvcParamKey | NAME        | Meaning                | Reference       |
-| ----------- | ------      | ---------------------- | --------------- |
-| 0           | key0        | Reserved               | (This document) |
-| 1           | alpn        | ALPN for alternative   | (This document) |
-|             |             | service                |                 |
-| 2           | port        | Port for alternative   | (This document) |
-|             |             | service                |                 |
-| 3           | esnikeys    | ESNI keys literal      | (This document) |
-| 65280-65534 | keyNNNNN    | Private Use            | (This document) |
-| 65535       | key65535    | Reserved               | (This document) |
+| SvcParamKey | NAME        | Meaning                       | Reference       |
```

Comment:
Do we want a column for "Repeats" to indicate whether it is allowed to repeat?  (Or maybe just leave that with the definition?)